gulpfile.js

```diff
@@ -19,7 +19,8 @@ var PATHS = {
   specs: 'src/**/*.spec.ts',
   demo: 'demo/**/*.ts',
   demoDist: 'demo/dist/**/*',
-  typings: 'typings/browser.d.ts'
+  typings: 'typings/browser.d.ts',
+  demoDocsJson: 'demo/src/docs.json'
```

Comment:
this is ok for now - we should either .gitignore this or move to a directory that's ignored so git doesn't keep pestering us that it's untracked.